extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduWriter.java

```diff
@@ -49,7 +50,9 @@ public synchronized void write(GeoWaveRow row) {
           kuduRow.populatePartialRow(insert.getRow());
         }
         OperationResponse resp = session.apply(insert);
-        if (resp.hasRowError()) {
+        if (resp != null && resp.hasRowError()) {
+          LOGGER.error("Encountered error while applying insert: {}", resp.getRowError());
+        } else if (session.getPendingErrors().getRowErrors().length > 0) {
           LOGGER.error("Encountered error while applying insert: {}", resp.getRowError());
```

Comment:
This should be looping through the row errors and printing each one, rather than using the `OperationResponse`